libs/verified-pages/src/pages/bulk-unsubscribe/en.ts

```diff
@@ -0,0 +1,29 @@
+export const en = {
+  bulkUnsubscribeTitle: "Bulk unsubscribe",
+  bulkUnsubscribeHeading: "Bulk unsubscribe",
+
+  // Tabs
```

Comment:
Not sure if we really need comments in language resource files. They are not in language files in other pages.

Reply:
Done :)